code/game/gamemodes/cult/talisman.dm

```diff
@@ -328,9 +328,20 @@
 		if(target.use(25))
 			new /obj/structure/constructshell(T)
 			user << "<span class='warning'>The talisman clings to the metal and twists it into a construct shell!</span>"
+			user << sound('sound/effects/magic.ogg',0,1,25)
+			PoolOrNew(/obj/effect/overlay/temp/cult/turf/open/floor, T)
 			qdel(src)
-		else
-			user << "<span class='warning'>The talisman requires at least 25 sheets of metal!</span>"
+	if(proximity_flag && istype(target, /obj/item/stack/sheet/plasteel) && iscultist(user))
```

Comment:
ok, ill still need to change istype from istype(target) to istype(target, blah/blah/metal)